tests/lifecycle/mssql_cases_test.go

```diff
@@ -23,6 +25,47 @@ func getMssqlCases(
 		return nil, err
 	}
 
+	provisionServerOnly := func() (*service.Instance, error) {
+
+		ctx, cancel := context.WithTimeout(context.Background(), time.Minute*20)
+		defer cancel()
+
+		s := serviceLifecycleTestCase{
```

Comment:
this is repeated code from below when the tests are created. I debated refactoring the test declarations out so this one could be reused, but this seemed more readable in the end.

Reply:
I've heard it suggested more than once before that in tests, readability is more important than avoiding DRY exceptions. 👍